lightkurve/utils.py

```diff
@@ -511,3 +511,40 @@ def detect_filetype(header):
     # if one of them is Undefined we expect `.lower()` to yield an AttributeError.
     except (KeyError, AttributeError):
         return None
+
+
+def boolean_mask_to_bitmask(aperture_mask):
```

Comment:
This function is only used in `lightcurvefile.py` and there are no use cases that require end users to call this function, at least none that I can think of right away?
For these reasons, I wonder if we want to move this function to `lightcurvefile.py` and make it a hidden function.  End users would still be able to call this function if they *really* wanted to, but I think our public API will be easier for users to navigate if we hide functions like this from the visible API.  Thoughts?

Reply:
I guess I meant `lightcurve.py`. It should probably just be a hidden function there?